onecodex/cli.py

```diff
@@ -152,20 +156,70 @@ def samples(ctx, samples):
 
 # utilites
 @onecodex.command('upload')
-@click.option("--no-threads", is_flag=True,
+@click.option('--no-threads', is_flag=True,
               help=OPTION_HELP['threads'],
               default=False)
-@click.option("--max-threads", default=4,
-              help=OPTION_HELP['max_threads'], metavar="<int:threads>")
-@click.argument('files', nargs=-1, required=False,
-                type=click.Path(exists=True))
+@click.option('--max-threads', default=4,
+              help=OPTION_HELP['max_threads'], metavar='<int:threads>')
+@click.argument('files', nargs=-1, required=False, type=click.Path(exists=True))
+@click.option('--clean', is_flag=True, help=OPTION_HELP['clean'], default=False)
+@click.option('--interleave/--no-interleave', is_flag=True, help=OPTION_HELP['interleave'],
```

Comment:
I just used the format from the `click` example; `--do-not-interleave` is a little long for my taste...